src/Kconfig

```diff
@@ -62,6 +62,13 @@ config GOLIOTH_MAX_NUM_OBSERVATIONS
     help
         The maximum number of CoAP paths which can be simultaneously observed.
 
+config GOLIOTH_BLOCKWISE_DOWNLOAD_BUFF_SIZE
```

Comment:
Please rename `s/BUFF/BUFFER/`. I think it will be more readable that way, since either `buf` or `buffer` are used as a name of a buffer.